docs/ispc.rst

```diff
@@ -2693,6 +2693,36 @@ the section `Logical and Selection Operations`_, which introduces helper
 functions in the standard library that provide these operations without
 short-circuiting.
 
+ISPC also enables optimizations for signed integer expressions, like in C/C++,
+where the compiler can rely on undefined behavior for signed overflow.
+
+Consider the following program:
+
+::
+
+    const uniform bool isGreater = (INT_MAX + 1) > INT_MAX;
+
+If computing with unbounded integers in addition, subtraction, and
+multiplication expressions, the value ``isGreater`` can safely be
```

Comment:
Will reword to make this less confusing with respect to the example program.

Reply:
```suggestion
If computing with unbounded integers, the value ``isGreater`` can safely be
```

Reply:
I don't like this write up of the problem. It contains crucial imprecision in the wording. I'll contribute documentation for this, please focus on finishing up the feature itself for now.